iree/test/e2e/xla_ops/partial/add_f16.mlir

```diff
@@ -0,0 +1,7 @@
+func @tensor() attributes { iree.module.export } {
```

Comment:
nit: also not sure how we name these in other tests, but `func @add_f16()` or `func @test_add_f16()` would be more descriptive

Reply:
I think we sometimes just let the file name stand on its own and make the function names describe the specific variant of the test (like "big tensor" or whatever). With such a simple case, that makes it hard to come up with a name :-D